apps/ehr/tests/e2e/specs/in-person/procedures.spec.ts

```diff
@@ -0,0 +1,222 @@
+import { Page, test } from '@playwright/test';
+import { DateTime } from 'luxon';
+import { CssHeader } from 'tests/e2e/page/CssHeader';
+import { openDocumentProcedurePage } from 'tests/e2e/page/DocumentProcedurePage';
+import {
+  InPersonProgressNotePage,
+  openInPersonProgressNotePage,
+} from 'tests/e2e/page/in-person/InPersonProgressNotePage';
+import { expectProceduresPage, openProceduresPage } from 'tests/e2e/page/ProceduresPage';
+import { ResourceHandler } from 'tests/e2e-utils/resource-handler';
+
+const resourceHandler = new ResourceHandler(`documentProceduresPage-mutating-${DateTime.now().toMillis()}`);
+const WOUND_CARE_PROCEDURE_TYPE = 'Wound Care / Dressing Change';
+const SPLINT_APPLICATION_PROCEDURE_TYPE = 'Splint Application / Immobilization';
+const X_RAY_CPT_CODE = '73000';
+const INJECTION_CPT_CODE = '11900';
+const X_RAY_CPT_NAME = 'X-ray of collar bone';
+const INJECTION_CPT_NAME = 'Injection into skin growth, 1-7 growths';
+const B12_DIAGNOSIS_CODE = 'D51.0';
+const FEVER_DIAGNOSIS_CODE = 'R50.9';
+const B12_DIAGNOSIS_NAME = 'Vitamin B12 deficiency anemia due to intrinsic factor deficiency';
+const FEVER_DIAGNOSIS_NAME = 'Fever, unspecified';
+const HEALTH_CARE_STUFF = 'Healthcare staff';
+const BOTH = 'Both';
+const TOPICAL = 'Topical';
+const LOCAL = 'Local';
+const ARM = 'Arm';
+const FACE = 'Face';
+const LEFT = 'Left';
+const NOT_APPLICABLE = 'Not Applicable';
+const STERILE = 'Sterile';
+const CLEAN = 'Clean';
+const SPLINT = 'Splint';
+const SPECULUM = 'Speculum';
+const PROCEDURE_DETAILS = 'test details';
+const PROCEDURE_DETAILS_2 = 'test details_edited';
+const YES = 'Yes';
+const NO = 'No';
+const BLEEDING = 'Bleeding';
+const ALLERGIC_REACTION = 'Allergic Reaction';
+const STABLE = 'Stable';
+const IMPROVED = 'Improved';
+const RETURN_IF_WORSENING = 'Return if worsening';
+const WOUND_CARE = 'Wound Care';
+const LESS_5_MIN = '< 5 min';
+const MORE_30_MIN = '> 30 min';
+const PROVIDER = 'Provider';
+const HEALTHCARE_STAFF = 'Healthcare staff';
+
+test.describe('Document Procedures Page mutating tests', () => {
+  test.beforeEach(async ({ page }) => {
+    if (process.env.INTEGRATION_TEST === 'true') {
+      await resourceHandler.setResourcesFast();
+    } else {
+      await resourceHandler.setResources();
+      await resourceHandler.waitTillAppointmentPreprocessed(resourceHandler.appointment.id!);
+    }
+    await setupPractitioners(page);
+  });
+
+  test.afterEach(async () => {
+    await resourceHandler.cleanupResources();
+  });
+
+  test('Fill and save all values on Document Procedures Page, values are saved and updated successfully- Happy path', async ({
```

Comment:
i've started appreciating the way [we have unique identifiers](https://github.com/masslight/ottehr/pull/3693/files#diff-f6fcaa1b93529267386b86445d7d587007322a337ac0a781874653598642d9c4R38) for e2e tests. this will help with communication when, after they've been split it up, we can reference which ones break

Reply:
this would still be appreciated. when you break it into steps, they could still be eg document procedure page 1 step 1 (DPP-1.1) or something